src/controllers/change.answers.controller.ts

```diff
@@ -0,0 +1,19 @@
+import {NextFunction, Request, Response} from "express";
+import {addToObjectionSession} from "../services/objection.session.service";
+import {Session} from "ch-node-session-handler";
+import {STRIKE_OFF_OBJECTIONS} from "../model/page.urls";
+import {CHANGE_ANSWER_KEY} from "../constants";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  const session: Session | undefined = req.session as Session;
+  if (session) {
+    const query: string | undefined = req.query["changePage"] as string;
+    if (query) {
+      addToObjectionSession(session, CHANGE_ANSWER_KEY, true);
+      const url: string = STRIKE_OFF_OBJECTIONS + "/" + query;
+      return res.redirect(url);
+    }
+    return next(new Error("No query present"));
```

Comment:
Please see comment above - something like "No page name present to change to" would make more sense as the log message I think.

Reply:
Updated message.